swap_meet/clothing.py

```diff
@@ -1,2 +1,8 @@
-class Clothing:
-    pass
\ No newline at end of file
+from .item import Item
+class Clothing(Item):
+    def __init__(self, condition = 0): # default arguments
+        super().__init__(category = "Clothing", condition = condition) # keyword arguments
```

Comment:
👍  great job hardcoding the `category` param inside the `super`. That way a user can't accidentally or intentionally change the attribute!